doc/related_projects.rst

```diff
@@ -50,6 +50,12 @@ enhance the functionality of scikit-learn's estimators.
   includes a replacement for ``GridSearchCV`` or ``RandomizedSearchCV`` to do
   cross-validated parameter search using any of these strategies.
 
+- `Neuraxle <https://github.com/Neuraxio/Neuraxle>`_
+  A library for building neat pipelines, providing the right abstractions to both
+  ease research, development, and deployment of your machine learning applications.
+  Compatible with deep learning frameworks, you can stream minibatches, use data
```

Comment:
Maybe you say here
"""
Compatible with deep learning frameworks and the scikit-learn API
"""
then?